test/index.spec.js

```diff
@@ -281,6 +282,36 @@ describe('desciple-core-api', () => {
       expect(exportedData[ssid.did][1]).to.deep.equal({ [claimlink2]: { 'need': 'wine' } })
       expect(exportedData[ssid.did][2]).to.deep.equal({ [attestationLink]: { 'agree': { [ssid2.did]: [ { [claimlink3]: { 'need': 'water' } } ] } } })
     })
+
+    it('should be able to import multiple verifiable claims in multiple channels in order (in ephemeral connector)', async () => {
+      let ssid = await discipl.newSsid('ephemeral')
+      let ssid2 = await discipl.newSsid('ephemeral')
+
+      await discipl.claim(ssid, { 'need': 'food' })
+      await discipl.claim(ssid, { 'match': 'link' })
+      await discipl.claim(ssid, { 'allow': 'some' })
+      await discipl.claim(ssid2, { 'require': 'drink' })
+      await discipl.claim(ssid2, { 'solved': 'problem' })
+      await discipl.claim(ssid2, { 'attendTo': 'wishes' })
+      let ld = await discipl.exportLD(ssid)
+      let ld2 = await discipl.exportLD(ssid2)
+
+      // reset ephemeral connector (in memory mode)
+      let ConnectorModuleClass = await loadConnector('ephemeral')
+      discipl.registerConnector('ephemeral', new ConnectorModuleClass())
+
+      let result = await discipl.importLD({ ...ld, ...ld2 })
+      expect(result).to.equal(true)
+
+      let channelData = await discipl.exportLD(ssid)
+      expect(channelData[ssid.did][0][Object.keys(channelData[ssid.did][0])]).to.deep.equal({ 'need': 'food' })
```

Comment:
I would consider maybe testing the entire data structure in one go. This is quite hard to read.

Reply:
ok, weĺl need to improve on how to handle these JSON exports to get more readable code, not only within tests in which you can check against the whole datastructure.